bubble_sort.rb

```diff
@@ -0,0 +1,55 @@
+def bubble_sort(array)
+  is_sorted = false
+
+  i = 0
+  length = array.length - 1
+
+  loop do
+    j = 0
+    is_sorted = true
+
+    break if i == length
+
+    loop do
+      break if j + i == length
+
+      if array[j] > array[j + 1]
+        array[j], array[j + 1] = array[j + 1], array[j]
+        is_sorted = false
+      end
+      break if j == length - 1
+
+      j += 1
+    end
+    break if is_sorted
+
+    i += 1
+  end
+  array.to_s
```

Comment:
The return value should be the sorted array and not a string

Reply:
**Noted**